WalletWasabi.Backend/Controllers/WabiSabiController.cs

```diff
@@ -119,7 +119,7 @@ public async Task ReadyToSignAsync(ReadyToSignRequestRequest request, Cancellati
 	/// Information about the current Rounds designed for the human eyes.
```

Comment:
The human monitor is for humans, not robots. The info must got to the `GetStatus`.